tests/conftest.py

```diff
@@ -98,6 +105,21 @@ def sites(db_session, clients):
     return sites
 
 
+@pytest.fixture()
+def inverters(db_session, sites):
+    """Create some fake inverters for site 0"""
+    inverters = []
+    num_inverters = 3
+    for j in range(num_inverters):
+        inverter = InverterSQL(site_uuid=sites[0].site_uuid, client_id=f"id{j+1}")
+        inverters.append(inverter)
```

Comment:
Nitpicking here, so very optional (current implementation is fine), but this could be a bit more succinct: 

```python
inverters = [
  InverterSQL(site_uuid=sites[0].site_uuid, client_id=f"id{j+1}")
  for j in range(num_inverters)
]
```